lib/broc-component-css-preprocessor.js

```diff
@@ -92,7 +93,21 @@ BrocComponentCssPreprocessor.prototype.write = function (readTree, destDir) {
         var podGuid = podPath.join('--') + '-' + guid();
         var fileContents = fs.readFileSync(path.join(srcDir, filepath)).toString();
 
-        buffer.push(processors[pod.extension](fileContents, podGuid));
+        if(pod.extension === 'scss') {
+          var outFilepath = filepath;
+          // TODO: introduce a useComponentNameAsFileName option which defaults to false
+          if (true) {
+            outFilepath = path.join('pod-styles', podPath.join('/') + '.' + pod.extension);
+          }
+
+          mkdirp.sync(path.join(destDir,path.dirname(outFilepath)));
```

Comment:
Why not just use `fs.makedirSync`? That way we don't have to add another dependency.